test/isTabbable.jest.test.js

```diff
@@ -0,0 +1,278 @@
+const { getByTestId, getByText } = require('@testing-library/dom');
+const { isTabbable } = require('../src/index.js');
+
+describe('isTabbable', () => {
+  it('throws an error if no node is provided', () => {
+    expect(() => isTabbable()).toThrow();
+  });
+
+  describe('returns true', () => {
+    it('returns true for a `button` element', () => {
+      const container = document.createElement('div');
+      container.innerHTML = '<button>Click me</button>';
+
+      expect(isTabbable(getByText(container, 'Click me'))).toBe(true);
+    });
+
+    it('returns true for an `input` element', () => {
+      const container = document.createElement('div');
+      container.innerHTML = '<input data-testid="testInput" />';
+
+      expect(isTabbable(getByTestId(container, 'testInput'))).toBe(true);
+    });
+
+    it('returns true for a `select` element', () => {
+      const container = document.createElement('div');
+      container.innerHTML = `<select data-testid="testSelect">
+          <option value="foo">foo</option>
+        </select>`;
+
+      expect(isTabbable(getByTestId(container, 'testSelect'))).toBe(true);
+    });
+
+    it('returns true for a `textarea` element', () => {
+      const container = document.createElement('div');
+      container.innerHTML = '<textarea data-testId="testTextarea"></textarea>';
+
+      expect(isTabbable(getByTestId(container, 'testTextarea'))).toBe(true);
+    });
+
+    it('returns true for an `a` anchor element with an `href` attribute', () => {
+      const container = document.createElement('div');
+      container.innerHTML =
+        '<a href="https://github.com/focus-trap/tabbable">Tabbable</a>';
+
+      expect(isTabbable(getByText(container, 'Tabbable'))).toBe(true);
+    });
+
+    // The README says `isTabbable` should return `true for:
+    // <a> elements with href or xlink:href attributes
+    // But this is returning `false`.
+    // Should the README be updated to not include `xlink:href` here?
+    it.skip('returns true for an `a` anchor element with an `xlink:href` attribute', () => {
```

Comment:
Same here. I need to confirm this behavior manually in a real app, outside of a test suite.